changelog.d/12254.doc

```diff
@@ -0,0 +1 @@
+Updated the doc with instructions on how to use free turn server from open relay project with Synapse
```

Comment:
Thanks for this! Looks like the CI is failing due to a lack of punctuation here. Also generally changelog entries take the imperative form:
```suggestion
Update turn docs with instructions on how to use a free turn server from open relay project with Synapse.
```